core/src/system_monitor_service.rs

```diff
@@ -702,6 +813,11 @@ impl SystemMonitorService {
                 ("total_num_threads", info.num_threads as i64, i64),
             )
         }
+
+        #[cfg(any(target_arch = "x86_64", target_arch = "x86_64"))]
+        unsafe {
```

Comment:
I _think_ we can move the `unsafe` block inside `report_cpuid_values()`, so it doesn't need to leak up to this otherwise safe block